commit 542160f0b6a7c26758c9575a8876f6624a5dd65f
Author: Girish Joshi <girish946@gmail.com>
Date:   Mon Mar 2 15:19:29 2020 -0500

    Fixed typo in run_command_array() in support/shell-container.c
    
    https://sourceware.org/bugzilla/show_bug.cgi?id=23991

diff --git a/support/shell-container.c b/support/shell-container.c
index 9bd90d3f60529079..e87ac5cf1baa84e5 100644
--- a/support/shell-container.c
+++ b/support/shell-container.c
@@ -228,7 +228,7 @@ run_command_array (char **argv)
       if (new_stderr != 2)
 	{
 	  dup2 (new_stderr, 2);
-	  close (new_stdout);
+	  close (new_stderr);
 	}
 
       if (builtin_func != NULL)
